tests/python/gaia-ui-tests/gaiatest/resources/Mozhome.html

```diff
@@ -0,0 +1,1061 @@
+   7/ / /~- \_\\      '-._     .-'                      /       //
+  ./ ( /-~-/||'=.__  '::. '-~'' {             ___   /  //     ./{
+ V   V-~-~| ||   __''_   ':::.   ''~-~.___.-'' _/  // / {_   /  {  /
+  VV/-~-~-|/ \ .'__'. '.    '::                     _ _ _        ''.
+  / /~~~~||VVV/ /  \ )  \        _ __ ___   ___ ___(_) | | __ _   .::'
+ / (~-~-~\\.-' /    \'   \::::. | '_ ` _ \ / _ \_  / | | |/ _` | :::'
+/..\    /..\__/      '     '::: | | | | | | (_) / /| | | | (_| | ::'
+vVVv    vVVv                 ': |_| |_| |_|\___/___|_|_|_|\__,_| ''
+
+Hi there, nice to meet you!
+
+Interested in having a direct impact on hundreds of millions of users? Join
+Mozilla, and become part of a global community that’s helping to build a
+brighter future for the Web.
+
+Visit https://careers.mozilla.org to learn about our current job openings.
+Visit https://www.mozilla.org/contribute for more ways to get involved and
+help support Mozilla.-->
+
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+    <!-- validates bing webmaster tools -->
+<meta name="msvalidate.01" content="B7B177115A634927D608514DA17B2574">
+<!-- YouTube Verification -->
+<meta name="google-site-verification" content="Mef8xJZ1-faaN2AdGdCfRFo5VeoXOSxjXNIzMsxcx94">
+
+    <title>We’re building a better Internet — Mozilla</title>
+    <meta name="description" content="Did you know? Mozilla — the maker of Firefox — fights to keep the Internet a global public resource open and accessible to all.">
+    <meta property="og:type" content="website">
+    <meta property="og:site_name" content="Mozilla">
+    <meta property="og:locale" content="en_US">
+    <meta property="og:url" content="https://www.mozilla.org/en-US/">
+    <meta property="og:image" content="https://mozorg.cdn.mozilla.net/media/img/home/page-image.3af4522ff5e7.png">
+    <meta property="og:title" content="We’re building a better Internet">
+    <meta property="og:description" content="Did you know? Mozilla — the maker of Firefox — fights to keep the Internet a global public resource open and accessible to all.">
+    <meta property="fb:page_id" content="262134952380">
+    <meta name="twitter:card" content="summary">
+    <meta name="twitter:site" content="@mozilla">
+    <meta name="twitter:domain" content="mozilla.org">
+    <meta name="twitter:app:name:googleplay" content="Firefox">
+    <meta name="twitter:app:id:googleplay" content="org.mozilla.firefox">
+    <link rel="author" type="text/plain" href="https://www.mozilla.org/humans.txt">
+    <link rel="apple-touch-icon" type="image/png" sizes="180x180" href="https://mozorg.cdn.mozilla.net/media/img/favicon/apple-touch-icon-180x180.00050c5b754e.png">
+    <link rel="icon" type="image/png" sizes="196x196" href="https://mozorg.cdn.mozilla.net/media/img/favicon/favicon-196x196.cb79ae3b6daf.png">
+    <link rel="shortcut icon" href="https://mozorg.cdn.mozilla.net/media/img/favicon.52506929be4c.ico">
```

Comment:
There's a lot of external resources listed in this file. Ideally something served by the local webserver should avoid any external resources. Otherwise, we might just as well use an external site in the first place.